jinja2/filters.py

```diff
@@ -75,6 +75,8 @@ def make_attrgetter(environment, attribute, postprocess=None):
     def attrgetter(item):
         for part in attribute:
             item = environment.getitem(item, part)
+            if type(item) == Undefined and default:
```

Comment:
`isinstance(Undefined)` (or `is Undefined` if we actually care about exact match).

Reply:
good catch, I don't think we actually care about an exact match. Updated in most recent commit.